config/locales/en.yml

```diff
@@ -40,7 +40,8 @@ en:
     edit:
       header: 'Edit your Team'
       legend: 'New Team Information'
-      note: 'All fields are required.'
+      required_fields_notice_1: 'Note: Fields marked with'
+      required_fields_notice_2: 'can not be blank.'
```

Comment:
Instead of using these 2 lines can we use the `_html` trick to take care of this?

```required_fields_notice_html: 'Note: Fields marked with <span style="color: red;">*</span> can not be blank.'```

Also, can be de-duplicate the 2 different spots in I18n where we have this string? Generally I18n is used for ease of translation between languages, so having the same string in 2 locations just makes the job harder for someone trying to translate the whole scoreboard.